features/satisfaction_report.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Satisfaction
+  As a Pension Wise data analyst
```

Comment:
We've got [Pension Wise data analyst](https://github.com/guidance-guarantee-programme/reporting/blob/master/features/where_did_hear_about_us_report.feature#L2) and [data analyst](https://github.com/guidance-guarantee-programme/reporting/blob/master/features/call_volumes_report.feature#L2) across reporting here, are the same thing or intentionally different?